pkg/networkservice/chains/nsmgr/unix_test.go

```diff
@@ -14,13 +14,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !windows
+// +build linux
```

Comment:
why the switch to only linux?  This precludes testing on the Mac OS... what was the driver?

Reply:
Thanks, Ed!

This was a code change suggested during review and I somehow missed that it would remove tests for MacOS.
I'm going to revert this back to `!windows`.

@denis-tingaikin , please, share your thoughts in case you have any objections.

Reply:
@edwarnicke This tag was incorrect. All tests actually check OS name on `linux`. So  I suggested just changing the build tag instead of using skip on non-linux os names in tests...

Reply:
@edwarnicke @denis-tingaikin , shall I check the tests with `!windows` on MacOS (with removed OS name check)?

Reply:
@Sol-0 Please do not use `!windows` with removed OS name check. Note: these tests are testing grpcfd library. The library required linux...

FYI: Test will work, but for MacOS they will not make sense because we are using null server/clients instead of server/clients based on grpcfd. for non-linux systems...

Reply:
@denis-tingaikin , ok, thanks, Denis! I'll revert the change.

Reply:
See at an example of implementation grpcfd based chain element for non-linux systems https://github.com/networkservicemesh/sdk/tree/c57ed95db17ead5a8c91d289fd70aa48084ad82e/pkg/registry/common/recvfd